scipy/cluster/tests/test_hierarchy.py

```diff
@@ -200,6 +200,22 @@ def test_linkage_cophenet_tdist_Z_Y(self, xp):
         xp_assert_close(c, expectedc, atol=1e-10)
         xp_assert_close(M, expectedM, atol=1e-10)
 
+    def test_gh_22183(self):
```

Comment:
Omission of the `xp` argument was intentional to signal that this is a patch scoped to NumPy only, even more strongly than the `cpu_only` directive above, and I'm not really wanting to wade into array API matters here for now.